packages/pds-ember/app/styles/pds/theme/_light.scss

```diff
@@ -1,277 +1,79 @@
+  --pds-shadow--highest: 0 6px 6px #{$_black--a06}, 0 12px 16px #{$_black--a20};
 }
-$borderColor: color.$ui-gray-300;
-$borderColor--l1: color.$ui-gray-200;
-$borderRadius: var(--pds-borderRadius);
-
-///----------------------------------------///
-/// CORE ACTION
-///----------------------------------------///
-$action-l2:   color.$blue-010; // Lightest (+2)
-$action-l1:   color.$blue-100; // Light (+1)
-$action-base: color.$blue-500; // Base (0)
-$action-d1:   color.$blue-700; // Dark (-1)
-
-
-///----------------------------------------///
-/// CORE STATUS (INFO)
-///----------------------------------------///
-$info-l2:   color.$blue-010; // Lightest (+2)
-$info-l1:   color.$blue-100; // Light (+1)
-$info-base: color.$blue-500; // Base (0)
-$info-d1:   color.$blue-700; // Dark (-1)
-
-
-///----------------------------------------///
-/// CORE STATUS (ALERT)
-///----------------------------------------///
-$alert-l2:   color.$orange-010; // Lightest (+2)
-$alert-l1:   color.$orange-100; // Light (+1)
-$alert-base: color.$orange-500; // Base (0)
-$alert-d1:   color.$orange-800; // Dark (-1)
-
 
-///----------------------------------------///
-/// CORE STATUS (SUCCESS)
-///----------------------------------------///
-$success-l2:   color.$green-010; // Lightest (+2)
-$success-l1:   color.$green-100; // Light (+1)
-$success-base: color.$green-500; // Base (0)
-$success-d1:   color.$green-800; // Dark (-1)
-
-
-///----------------------------------------///
-/// CORE STATUS (WARNING)
-///----------------------------------------///
-$warning-l2:   color.$yellow-010; // Lightest (+2)
-$warning-l1:   color.$yellow-100; // Light (+1)
-$warning-base: color.$yellow-500; // Base (0)
-$warning-d1:   color.$yellow-800; // Dark (-1)
-
-
-///----------------------------------------///
-/// CORE STATUS (FAILURE)
-///----------------------------------------///
-// TODO: rename to "error" instead of "failure"
-$failure-l2:   color.$red-010; // Lightest (+2)
-$failure-l1:   color.$red-100; // Light (+1)
-$failure-base: color.$red-500; // Base (0)
-$failure-d1:   color.$red-800; // Dark (-1)
-
-
-///----------------------------------------///
-/// SHADOWS
-///----------------------------------------///
-// Focus
-$boxShadow--focus: 0 0 0 3px #{rgba(color.$blue-500, 0.5)};
-// Elevation
-$_black--a05: rgba(0, 0, 0, 0.05);
-$_black--a06: rgba(0, 0, 0, 0.06);
-$_black--a08: rgba(0, 0, 0, 0.08);
-$_black--a10: rgba(0, 0, 0, 0.10);
-$_black--a20: rgba(0, 0, 0, 0.20);
-$_black--a25: rgba(0, 0, 0, 0.25);
-$boxShadow--well: inset 0 3px 1px $_black--a06;
-$boxShadow--low:     0 4px 4px $_black--a25;
-$boxShadow--middle:  0 4px 8px $_black--a05, 0 4px  4px $_black--a10;
-$boxShadow--high:    0 5px 5px $_black--a08, 0 8px  10px $_black--a10;
-$boxShadow--highest: 0 6px 6px $_black--a06, 0 12px 16px $_black--a20;
-
-$SHADOWS: (
-  well: $boxShadow--well,
-  low: $boxShadow--low,
-  middle: $boxShadow--middle,
-  high: $boxShadow--high,
-  highest: $boxShadow--highest,
-);
```

Comment:
Shadows are no longer hard-coded as Sass members.  Each theme will define the same set of shadows via `--pds-theme--*` CSS custom properties and `getShadow()` has been replaced with `boxShadow()` to simplify the ad hoc application of various `box-shadow` definitions.